[2022-10-27 22:48:50,743] {processor.py:153} INFO - Started process (PID=53827) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:48:50,745] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:48:50,749] {logging_mixin.py:115} INFO - [2022-10-27 22:48:50,748] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:48:51,190] {logging_mixin.py:115} INFO - [2022-10-27 22:48:51,090] {dagbag.py:320} ERROR - Failed to import: /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
Traceback (most recent call last):
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py", line 15, in <module>
    tarefa_4 = BashOperator(
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 373, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2022-10-27 22:48:51,192] {processor.py:653} WARNING - No viable dags retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:48:51,794] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 1.067 seconds
[2022-10-27 22:49:33,345] {processor.py:153} INFO - Started process (PID=53912) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:49:33,366] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:49:33,366] {logging_mixin.py:115} INFO - [2022-10-27 22:49:33,366] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:49:33,379] {logging_mixin.py:115} INFO - [2022-10-27 22:49:33,377] {dagbag.py:320} ERROR - Failed to import: /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
Traceback (most recent call last):
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py", line 15, in <module>
    tarefa_4 = BashOperator(
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 373, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2022-10-27 22:49:33,380] {processor.py:653} WARNING - No viable dags retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:49:33,659] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.325 seconds
[2022-10-27 22:50:17,685] {processor.py:153} INFO - Started process (PID=53993) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:50:17,716] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:50:17,719] {logging_mixin.py:115} INFO - [2022-10-27 22:50:17,719] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:50:17,770] {logging_mixin.py:115} INFO - [2022-10-27 22:50:17,763] {dagbag.py:320} ERROR - Failed to import: /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
Traceback (most recent call last):
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py", line 15, in <module>
    tarefa_4 = BashOperator(
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 373, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2022-10-27 22:50:17,772] {processor.py:653} WARNING - No viable dags retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:50:18,031] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.365 seconds
[2022-10-27 22:51:03,310] {processor.py:153} INFO - Started process (PID=54080) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:51:03,323] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:51:03,325] {logging_mixin.py:115} INFO - [2022-10-27 22:51:03,325] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:51:03,349] {logging_mixin.py:115} INFO - [2022-10-27 22:51:03,348] {dagbag.py:320} ERROR - Failed to import: /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
Traceback (most recent call last):
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py", line 15, in <module>
    tarefa_4 = BashOperator(
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 373, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2022-10-27 22:51:03,350] {processor.py:653} WARNING - No viable dags retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:51:03,584] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.280 seconds
[2022-10-27 22:51:52,034] {processor.py:153} INFO - Started process (PID=54192) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:51:52,054] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:51:52,057] {logging_mixin.py:115} INFO - [2022-10-27 22:51:52,056] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:51:52,094] {logging_mixin.py:115} INFO - [2022-10-27 22:51:52,090] {dagbag.py:320} ERROR - Failed to import: /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
Traceback (most recent call last):
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py", line 15, in <module>
    tarefa_4 = BashOperator(
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 373, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2022-10-27 22:51:52,095] {processor.py:653} WARNING - No viable dags retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:51:52,759] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.739 seconds
[2022-10-27 22:52:49,427] {processor.py:153} INFO - Started process (PID=54279) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:52:49,430] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:52:49,434] {logging_mixin.py:115} INFO - [2022-10-27 22:52:49,433] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:52:49,455] {logging_mixin.py:115} INFO - [2022-10-27 22:52:49,453] {dagbag.py:320} ERROR - Failed to import: /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
Traceback (most recent call last):
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py", line 15, in <module>
    tarefa_4 = BashOperator(
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 373, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2022-10-27 22:52:49,456] {processor.py:653} WARNING - No viable dags retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:52:49,667] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.259 seconds
[2022-10-27 22:53:33,597] {processor.py:153} INFO - Started process (PID=54365) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:53:33,623] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:53:33,625] {logging_mixin.py:115} INFO - [2022-10-27 22:53:33,625] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:53:33,650] {logging_mixin.py:115} INFO - [2022-10-27 22:53:33,646] {dagbag.py:320} ERROR - Failed to import: /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
Traceback (most recent call last):
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py", line 15, in <module>
    tarefa_4 = BashOperator(
  File "/home/nayara/Documentos/airflowalura/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 373, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'bash_command'
[2022-10-27 22:53:33,651] {processor.py:653} WARNING - No viable dags retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:53:33,948] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.360 seconds
[2022-10-27 22:54:17,612] {processor.py:153} INFO - Started process (PID=54459) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:54:17,639] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:54:17,641] {logging_mixin.py:115} INFO - [2022-10-27 22:54:17,641] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:54:17,660] {processor.py:651} INFO - DAG(s) dict_keys(['meu_primeiro_dag']) retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:54:20,013] {logging_mixin.py:115} INFO - [2022-10-27 22:54:20,012] {manager.py:508} INFO - Created Permission View: can read on DAG:meu_primeiro_dag
[2022-10-27 22:54:21,107] {logging_mixin.py:115} INFO - [2022-10-27 22:54:21,106] {manager.py:508} INFO - Created Permission View: can delete on DAG:meu_primeiro_dag
[2022-10-27 22:54:21,661] {logging_mixin.py:115} INFO - [2022-10-27 22:54:21,660] {manager.py:508} INFO - Created Permission View: can edit on DAG:meu_primeiro_dag
[2022-10-27 22:54:21,662] {logging_mixin.py:115} INFO - [2022-10-27 22:54:21,661] {dag.py:2379} INFO - Sync 1 DAGs
[2022-10-27 22:54:21,870] {logging_mixin.py:115} INFO - [2022-10-27 22:54:21,870] {dag.py:2398} INFO - Creating ORM DAG for meu_primeiro_dag
[2022-10-27 22:54:21,957] {logging_mixin.py:115} INFO - [2022-10-27 22:54:21,957] {dag.py:2927} INFO - Setting next_dagrun for meu_primeiro_dag to 2022-10-27T00:00:00+00:00, run_after=2022-10-28T00:00:00+00:00
[2022-10-27 22:54:23,335] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 5.728 seconds
[2022-10-27 22:55:14,416] {processor.py:153} INFO - Started process (PID=54567) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:55:14,482] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:55:14,483] {logging_mixin.py:115} INFO - [2022-10-27 22:55:14,483] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:55:14,498] {processor.py:651} INFO - DAG(s) dict_keys(['meu_primeiro_dag']) retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:55:14,523] {logging_mixin.py:115} INFO - [2022-10-27 22:55:14,523] {dag.py:2379} INFO - Sync 1 DAGs
[2022-10-27 22:55:14,547] {logging_mixin.py:115} INFO - [2022-10-27 22:55:14,547] {dag.py:2927} INFO - Setting next_dagrun for meu_primeiro_dag to 2022-10-27T00:00:00+00:00, run_after=2022-10-28T00:00:00+00:00
[2022-10-27 22:55:14,943] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.532 seconds
[2022-10-27 22:56:05,826] {processor.py:153} INFO - Started process (PID=54661) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:56:05,829] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:56:05,832] {logging_mixin.py:115} INFO - [2022-10-27 22:56:05,832] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:56:05,871] {processor.py:651} INFO - DAG(s) dict_keys(['meu_primeiro_dag']) retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:56:05,902] {logging_mixin.py:115} INFO - [2022-10-27 22:56:05,902] {dag.py:2379} INFO - Sync 1 DAGs
[2022-10-27 22:56:05,931] {logging_mixin.py:115} INFO - [2022-10-27 22:56:05,931] {dag.py:2927} INFO - Setting next_dagrun for meu_primeiro_dag to 2022-10-27T00:00:00+00:00, run_after=2022-10-28T00:00:00+00:00
[2022-10-27 22:56:06,190] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.384 seconds
[2022-10-27 22:56:50,776] {processor.py:153} INFO - Started process (PID=54805) to work on /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:56:50,797] {processor.py:641} INFO - Processing file /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py for tasks to queue
[2022-10-27 22:56:50,801] {logging_mixin.py:115} INFO - [2022-10-27 22:56:50,801] {dagbag.py:507} INFO - Filling up the DagBag from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:56:50,829] {processor.py:651} INFO - DAG(s) dict_keys(['meu_primeiro_dag']) retrieved from /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py
[2022-10-27 22:56:50,854] {logging_mixin.py:115} INFO - [2022-10-27 22:56:50,854] {dag.py:2379} INFO - Sync 1 DAGs
[2022-10-27 22:56:50,882] {logging_mixin.py:115} INFO - [2022-10-27 22:56:50,882] {dag.py:2927} INFO - Setting next_dagrun for meu_primeiro_dag to 2022-10-27T00:00:00+00:00, run_after=2022-10-28T00:00:00+00:00
[2022-10-27 22:56:51,372] {processor.py:161} INFO - Processing /home/nayara/Documentos/airflowalura/dags/meu_primeiro_dag.py took 0.616 seconds
